### PR TITLE
Clean up environment variables

### PR DESCRIPTION
**Pull request by semtexzv**

### CODE REVIEW

File: scripts/wait-for-services.sh [hunk 1]
```diff
@@ -4,9 +4,9 @@ set -e
 
 cmd="$@"
 
-if [ ! -z "$POSTGRESQL_DATABASE" ]; then
+if [ ! -z "$DB_HOST" ]; then
   >&2 echo "Checking if PostgreSQL is up"
-  until PGPASSWORD="$POSTGRESQL_PASSWORD" psql -h "$POSTGRESQL_HOST" -p "$POSTGRESQL_PORT" -U "$POSTGRESQL_USER" -d "$POSTGRESQL_DATABASE" -c '\q' -q 2>/dev/null; do
```
Comment (Josca) [inline scripts/wait-for-services.sh]: It's broken. On `docker-compose up --build db platform`, I've got:
~~~
....
platform_1  | JSON generated: 29/30
platform_1  | JSON generated: 30/30
platform_1  | + ./wait-for-services.sh ./send-kafka-requests.sh
platform_1  | Checking if PostgreSQL is up
platform_1  | PostgreSQL is unavailable - sleeping
platform_1  | PostgreSQL is unavailable - sleeping
...
~~~

Comment (semtexzv) [reply scripts/wait-for-services.sh]: Fixed.

File: conf/common.env [hunk 2]
```diff
@@ -11,3 +10,17 @@ RETRY_INTERVAL=5
...
+DB_USER=admin
+DB_PASSWD=passwd
+DB_PORT=5432
+
+LISTENER_KAFKA_TOPIC=host.packages
+LISTENER_KAFKA_ADDRESS=platform:9092
+LISTENER_BUFFER_SIZE=10
+LISTENER_BATCH_WRITE=off
+
+BENCHMARK_MESSAGES=50
```
Comment (Josca) [inline conf/common.env]: As you increased msgs number from 30->50 I got this, did you too?
~~~
platform_1  | >>>>[2019-11-07 10:48:11,939] WARN [Producer clientId=console-producer] Got error produce response with correlation id 4 on topic-partition host.packages-0, retrying (2 attempts left). Error: NETWORK_EXCEPTION (org.apache.kafka.clients.producer.internals.Sender)
~~~
I've got this 4x.

Comment (semtexzv) [reply conf/common.env]: Yes, but the tests seem to be working, I'm rewriting the sending script to use single producer

Comment (Josca) [reply conf/common.env]: Not quite, it sends some messages multiple times. It's seen from items in database (<50). Set it back to 30, please. It's working without warns and these mentioned problems.

Comment (semtexzv) [reply conf/common.env]: Try now. I added python script to send the messages, so that the messages are sent over single producer.

**Pull request merged**